plugins/commander.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: commander
+spec:
+  homepage: https://github.com/schabrolles/kubectl-commander.git
+  shortDescription: interactive app for interacting with Kubernetes objects.
```

Comment:
* let's capitalize this sentence
* no need to say words like "app" or as they're obvious. Just say "Interactively manage Kubernetes resources"